Documentation/ApiOverview/CommandControllers/Tutorial.rst

```diff
@@ -0,0 +1,226 @@
+.. include:: /Includes.rst.txt
+
+.. _console-command-tutorial:
+
+========
+Tutorial
+========
+
+Create a console command from scratch
+=====================================
+
+A console command is always situated in an Extension. If you want to create
+one, :ref:`kickstart a custom extension <extension-kickstart>` or use your
+sitepackage extension.
+
+Creating a basic command
+========================
+
+The extension kickstarter ":ref:`Make <extension-make>`" offers a convenient
+console command that creates a new command in an extension of your choice:
+:ref:`Create a new console command with "Make" <extension-make-console-command>`.
+You can use "Make" to create a console command even if your extension was
+created by different means.
+
+This command can be found in the
+`Examples extension <https://github.com/TYPO3-Documentation/t3docs-examples>`.
+
+1. Register the command
+-----------------------
+
+Register via DI in :file:`Configuration/Services.yaml` by adding the service
```

Comment:
```suggestion
Register the command in :file:`Configuration/Services.yaml` by adding the service
```

As I understand, DI is injecting dependencies into classes. Here we register a command which is something different.

Reply:
I think you are true, it was already written like that